Sources/FluidAudio/DownloadUtils.swift

```diff
@@ -496,3 +509,62 @@ public class DownloadUtils {
         }
     }
 }
+
+// MARK: - eSpeak NG Data (G2P)
+
+extension DownloadUtils {
+    /// Ensure the eSpeak NG data bundle is available locally for the given repo (kokoro).
+    /// Downloads `Resources/espeak-ng-data.zip` from HuggingFace and extracts it.
+    /// - Returns: URL to the `espeak-ng-data` directory that contains `voices/`.
+    @available(macOS 13.0, iOS 16.0, *)
+    public static func ensureEspeakDataBundle(in modelsDirectory: URL) async throws -> URL {
+        let logger = AppLogger(category: "DownloadUtils")
+
+        let repo = Repo.kokoro
+        let repoPath = modelsDirectory.appendingPathComponent(repo.folderName)
+        let bundleRoot = repoPath.appendingPathComponent("Resources/espeak-ng/espeak-ng-data.bundle/espeak-ng-data")
```

Comment:
what if people want to manually download? They would just skip this entirely and manually copy the zip in before calling?